app/models/tests/test_test_group_model.py

```diff
@@ -111,6 +112,7 @@ def test_group_doc_to_dict(self):
             "kernel_image": "kernel-image",
             "lab_name": "lab-name",
             "mach": "mach",
+            "modules": "moduli",
```

Comment:
@gctucker any special reason to use this typo in test?

Reply:
Yes, to use an arbitrary value that is not likely to be used anywhere else otherwise.  This is to avoid having default values being set and hiding an issue.  It's also the plural of `modulus` :)